lib/omniauth/strategies/survey_monkey.rb

```diff
@@ -28,7 +28,8 @@ class SurveyMonkey < OmniAuth::Strategies::OAuth2
       option :client_options, {
         :site           => 'https://api.surveymonkey.net',
         :authorize_url  => '/oauth/authorize',
-        :token_url      => '/oauth/token'
+        :token_url      => '/oauth/token',
+        :auth_scheme => 'request_body'
```

Comment:
This doesn't change anything for our current main since we're still using oauth 1.X.X and that is a default value, but this allows us to upgrade to 2.X.X in follow on PR (default value is different in 2.X.X)